### PR TITLE
Guides folder

### PR DESCRIPTION
**Pull request by safronovD**
- Cluster preparation (TODO: new ssh keys)
- Deploy with Kubespray
- Provisioners

### CODE REVIEW

File: instructions/Helpful.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Cert-manager
```
Comment (fedor-chemashkin) [inline instructions/Helpful.md]: Please rename the ` instructions` folder to `guides`

File: instructions/Provisioners.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+## Local-path
```
Comment (fedor-chemashkin) [inline instructions/Provisioners.md]: Maybe we can call this file `storage` or `storage-provisioners`

File: instructions/Ssh-keys.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+## SSH
```
Comment (fedor-chemashkin) [inline instructions/Ssh-keys.md]: Maybe we can call this file `openstack` and put all related info there for now

Comment (safronovD) [reply instructions/Ssh-keys.md]: is "openstack" folder a good way?

Comment (fedor-chemashkin) [reply instructions/Ssh-keys.md]: @safronovD 
yes

**Review by fedor-chemashkin — APPROVED** [review]

**Pull request merged**